cmd/ingress-operator/start.go

```diff
@@ -118,12 +124,68 @@ func start() error {
 		os.Exit(1)
 	}
 
+	// Set up the channels for the watcher and operator.
+	stop := make(chan struct{})
+	signals := signals.SetupSignalHandler()
```

Comment:
I just realized that this variable declaration shadows the package name.  Maybe better use a distinct name for readability?

Reply:
I'll rename the channel to `signal`.